src/Web/Views/Shared/_Layout.cshtml

```diff
@@ -60,19 +62,26 @@
     <header class="navbar navbar-light navbar-static-top">
         <div class="container">
             <article class="row">
-                <section class="col-lg-6 col-md-5 col-xs-11">
+                <section class="col-lg-2 col-md-4 col-xs-11">
                     <a asp-area="" asp-controller="Catalog" asp-action="Index" class="navbar-brand">
                         <img src="~/images/brand.png" alt="Microsoft Fraud Protection Merchant Application" title="@apiBaseUrl" />
                     </a>
                 </section>
                 <partial name="_LoginPartial" />
+                @using (Html.BeginForm("Update", "Environment", FormMethod.Post, new { id = "environmentupdate" }))    
```

Comment:
In addition to this _Layout.cshtml, there is also one for the Admin view.  Do you want to add the same form there?  Otherwise, one might be set but it won't show.  Admitedly, we don't use that admin area often/even, but I think we should at least keep parity between the two areas.

Reply:
fixed, thanks for pointing it out.